src/components/VxpAccordionItem.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <StackLayout :class="item_class" :id="item_id">
```

Comment:
css class names should be corrected

Reply:
css class names updated wrt project predefined styles